lib/core-server/src/__snapshots__/cra-ts-essentials_manager-dev

```diff
@@ -216,3 +216,220 @@ Object {
   ],
 }
 `;
+
+exports[`cra-ts-essentials manager production mode 1`] = `
```

Comment:
🤔  I don't understand this change. Is it necessary?

Reply:
I don't understand this either :smile: I only changed the template file and tests. I think yarn test -u might be the culprit

Reply:
Please assist with the latest commit. I tried to revert this file hoping it would be removed from the PR but the new commit shows the deleted lines. Is it as simple as using "Delete file" from the menu in "Files changed" to remove it from the PR?

Reply:
Nevermind. I see it's been removed from Files changed